docs/visual/faq.md

```diff
@@ -0,0 +1,143 @@
+---
+id: faq
+title: Visual Testing FAQ
+sidebar_label: FAQ
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+## General
+
+### Can I revert a recently accepted snapshot?
+Yes. After accepting a snapshot, you can revert its acceptance before running another build:
+1. Go to the **State Detail** page.
+2. Click on the **Accepted** status dropdown
+3. Select **Revert**.<br/>
+   <img src={useBaseUrl('img/visual/revertAccepted.png')} alt="Revert accepted screenshot" width="150" />
+
+### How do I show lazy-loaded content in my screenshot?
+You need to trigger the lazy-loaded content so that it loads and displays in your web page. How to do this depends on how the lazy-loaded content is triggered to load.
+
+For example, if content is loaded based on scrolling down to a particular section of your web page, then you will need to scroll to that section using JavaScript:
+
+```js
+window.scroll(0, 1000);
+```
+
+### Why is my screenshot not full-page?
+Screener captures a full-page screenshot by default in all browsers. But the height of the full-page screenshot is based on the scrollable height of the main document body.
+
+If the body is not scrollable, then the screenshot captured will appear to be cut off. For example, if the "overflow" style on the body is set to "hidden", then the body cannot be scrolled.
+
+#### Are there workarounds?
+Yes, but the workaround depends on how the website CSS was built and understanding what is causing the body to not be scrollable.
+
+If the overflow style on the body is "hidden", then the client could execute the following JS in an Execute Script command in the test to set body overflow to `'auto'` before capturing a snapshot:
+
+```js
+document.body.style.overflow = 'auto'
+```
+
+
+## Visual E2E
+
+### What Sauce Labs browsers are supported by Visual E2E?
+Visual E2E is unified with Sauce VMs, and uses Sauce browsers exclusively. For a detailed list, see [Visual E2E Supported Browsers and Operation Systems](https://docs.saucelabs.com/visual/e2e-testing/supported-browsers).
```

Comment:
You changed the title to `operating systems`, do we need to match that here?

Reply:
Yes that's part of the PR (https://github.com/saucelabs/sauce-docs/pull/849/files). Thanks for checking!